src/sync/streaming/UpdateWorkers/__tests__/MySegmentsUpdateWorker.spec.ts

```diff
@@ -41,7 +42,7 @@ describe('MySegmentsUpdateWorker', () => {
     // setup
     const mySegmentsSyncTask = mySegmentsSyncTaskMock();
     Backoff.__TEST__BASE_MILLIS = 1; // retry immediately
-    const mySegmentUpdateWorker = MySegmentsUpdateWorker(mySegmentsSyncTask);
+    const mySegmentUpdateWorker = MySegmentsUpdateWorker(mySegmentsSyncTask, telemetryTrackerFactory());
```

Comment:
When possible, create a single instance to reuse. In this case, `const telemetryTracker = telemetryTrackerFactory()` can be placed after `mySegmentsSyncTaskMock` and reused in each test.